src/gpg/fn/print_key.py

```diff
@@ -0,0 +1,117 @@
+# Print the details of the given key id.
+import os
+import sys
+import textwrap
+import time
+
+import gpg
+
+
+def print_key(identity):
+    """
+    print the details of key whose identitiy is passed
+    """
+    c = gpg.Context()
+    key = list(c.keylist(identity, mode=gpg.constants.keylist.mode.SIGS))
+
+    # exit with status 1 if more than one key matches are found for identity
+    if len(key) > 1:
+        exit(1)
+    else:
+        key = key[0]
+
+    if os.environ["DEBUG"] == "yes":
+        print(key)
+        print()
```

Comment:
Empty print block. Use the `print` functions `end` parameter to specify newline.
`print(key, end="\n")`